pkg/tests/api/alerting/api_admin_configuration_test.go

```diff
@@ -29,7 +29,7 @@ import (
 func TestIntegrationAdminConfiguration_SendingToExternalAlertmanagers(t *testing.T) {
 	testinfra.SQLiteIntegrationTest(t)
 
-	const disableOrgID int64 = 3
+	const disableOrgID int64 = 2
```

Comment:
nit; could you fix this comment too:
https://github.com/grafana/grafana/blob/19a40e50ba31daaca2c2eab9896a10dfc8302fbd/pkg/tests/api/alerting/api_admin_configuration_test.go#L38